tests/common/file_utils.rs

```diff
@@ -15,9 +18,18 @@ use std::path::PathBuf;
 /// get_path_in_temp(&path_in_temp);
 ///
 pub fn get_path_in_temp(file_path: &str) -> PathBuf {
-    let temp_dir = Temp::new_dir().unwrap();
+    let temp_dir = mktemp::Temp::new_dir().unwrap();
     let mut path = temp_dir.to_path_buf();
     path.push(&file_path);
     temp_dir.release();
```

Comment:
I'd rather use crate `tempfile` as it is more mature. This is for tests, so it does not matter much as long as the test can clean up for itself.

Reply:
arf. I think my bad I suggested mktemp last time. And now looking at it, tempfile seems a better long term bet.